Fix Python version specifier (#62)
TIM-JYU/TIDE-CLI: README.md

- Python 3.10 or higher
- [Poetry](https://python-poetry.org/docs/#installation) and [Poetry shell plugin](https://github.com/python-poetry/poetry-plugin-shell) for dependency management, see the links for installation instructions

> - If you don't want to potentially break your system-controlled packages, use `pipx` to install Poetry, and skip installing the shell plugin:

Reviewer: Is this related to using pip install with system default Python? If so, I don't think mentioning the problem here is necessary since poetry docs do not have such installation instructions. Or is this a problem with [Poetry installation script](https://python-poetry.org/docs/#installing-with-the-official-installer)? If so, an issue to poetry would be a better option.

Also, is poetry shell somehow related to the system-controlled packages breaking, or is the problem the same as with poetry? It seems unclear here why should poetry shell not be installed. It could be worth mentioning that poetry shell installation is optional because of the `poetry env activate` alternative.

Author: Yeah, the wording could be clearer, I was in a bit of a hurry when I wrote it down.

The issue is two-fold, or rather, there are at least two problems. These may. or may not, be limited to Arch/Manjaro (I haven't tested other distros):

1) Currently, the packages [python-poetry](https://archlinux.org/packages/extra/any/python-poetry/) and [python-poetry-plugin-export](https://archlinux.org/packages/extra/any/python-poetry-plugin-export/) are essentially broken due to a cyclical dependency. So you either can't install them, or risk breaking your system packages. I haven't yet had the time to obtain an account to report this on Arch's GitLab.

2) Trying to install the `poetry-plugin-shell` will trigger Poetry to update the dependency `more-itertools`(and possibly others), which [fails](https://github.com/python-poetry/poetry-plugin-shell/issues/37).
Currently, at least Arch hasn't yet updated the packaging to the latest version (I've already flagged it as out-of-date, no idea when the package maintainer gets around to updating it). So one would probably need to compile the package themselves to sidestep this issue, but I'm not sure if this is the only package with this problem.

Since `pipx` solved [1], and `poetry env activate` solved [2], I did not try Poetry's install script. It wouldn't  solve [2] regardless.

Regarding `python-poetry-shell` I'm not quite sure why it should be installed at all, given that we don't seem to be using it for anything here. So I'm partial to mentioning it as a non-requirement, and favoring `poetry env activate` instead.

Reviewer: So is the problem related to the mentioned arch linux packages (which are not mentioned in the poetry docs so shouldn't be an issue if following this readme)?

Reviewer: poetry env activate does not work for me, so I don't think it should be the only option (I don't remember how I installed it though).

One way to go would be to just have `poetry run ...` instructions to keep things simple.

Author: The additional comments should now be a bit clearer.

Step 2: Modify src/tidecli/tide_config.py BASE_URL to correspond the TIM-server being used

> Step 3: Run `poetry shell` to enter virtual environment. Skip this step if you installed Poetry via **pipx** (see "Requirements" above).

Reviewer: I think it'd be easier to read if the alternative was mentioned here directly. 

```suggestion
Step 3: Run `poetry shell` to enter virtual environment  (or `poetry env activate` if not using the shell plugin). 
```

Author: I'm in favor of removing the instructions/requirement of installing the `poetry shell` altogether, but I agree your suggestion on the wording is clearer.
